src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -739,4 +739,150 @@ var fluid = fluid || fluid_2_0_0;
             (transformSpec.max === undefined ||  transformSpec.max >= value) ? true : false;
     };
 
+    /**
+     *
+     * Convert a string to a Boolean.
+     *
+     * The following are all `false`: `undefined`, `null`, "", "0", "false"
```

Comment:
Hi, @jobara.  Agreed on using JSON syntax rather than GfM to distinguish strings from non-strings.

As to the use case, this function was designed to help in relaying HTML form element values, which are always strings.

However, I agree that it's easy to imagine people stuffing other "falsy" material in.  I can see two ways to clarify.  One is to throw an error if something other than a string (or undefined/null) is passed.  The other (which seems less harsh) is to handle `false` and `0`, which I have done in the latest commit.